src/helpers/buildWalletSections.js

```diff
@@ -79,14 +79,61 @@ const filterWalletSections = sections =>
     data ? get(header, 'totalItems') : true
   );
 
+const addEth = section => {
+  const assets = store.getState().data.genericAssets;
+
+  if (assets.eth) {
+    const { relative_change_24h, value } = assets.eth.price;
+    const zeroEthRow = {
```

Comment:
can we move to it to some constant outside?

Reply:
i think this would be a good task for Skyler -- i think this constant should be generated by an `asset` factory that Skyler should build for us.. we can then use this factory to generate `asset` objects used for automated tests.

so im in favor of leaving this for now.

@osdnk im gonna merge this but if you have concerns we can revert!